proxy/http/HttpConnectionCount.h

```diff
@@ -85,6 +86,7 @@ class OutboundConnTrack
   // Unfortunately these are not used in RecordsConfig.cc so that must be made consistent by hand.
   // Note: These need to be @c constexpr or there are static initialization ordering risks.
   static constexpr std::string_view CONFIG_VAR_MAX{"proxy.config.http.per_server.connection.max"_sv};
+  static constexpr std::string_view CONFIG_VAR_MIN{"proxy.config.http.per_server.min_keep_alive_connections"_sv};
```

Comment:
If this name is going to be changed, why not make it more consistent, e.g.
```
proxy.config.http.per_server.connection.min
```

Reply:
> If this name is going to be changed, why not make it more consistent, e.g.
> 
> ```
> proxy.config.http.per_server.connection.min
> ```

Yea, good point.  Let me change the name (and update the docs) to be more consistent.